client/notifications/notification.js

```diff
@@ -55,6 +55,17 @@ Meteor.startup(function() {
 					KonchatNotification.newMessage(notification.payload.rid);
 				}
 			});
+
+			RocketChat.Notifications.onUser('subscriptions-changed', function(action, sub) {
```

Comment:
What happens when you was added while offline and open the chat after that in a new window? You will receive the subscriptions in another place, it will not be an change while logged in.

Reply:
I'm looking into it

Reply:
covered by dc0aa06651453e41a1cb58814ddb6418f80d2f7e =)